packages/appstore/src/components/cfd-account-manager/index.ts

```diff
@@ -0,0 +1,4 @@
+import CFDAccountManager from './cfd-account-manager';
+import './cfd-account-manager.scss';
```

Comment:
@hirad-rewok Just a question, Why do we import the CSS here and not inside the component itself? 🤔 If someone imports the file directly the style won’t load in this case.

Reply:
The structure that we have in our project is this way. We bundle our component with its styling file and export them in an index file. With this approach you do not need to specify the name of the jsx, tsx file that you need and just pointing to the folder of that component suffices. Some people (in other projects) do not use the index file and just import the component by pointing to the exact file that they need but I think we have a cleaner approach this way and it's easier for configuring our module bundler as well.

Reply:
@hirad-rewok Thank you for your explanation 🌹 Although my question wasn’t regarding the barrel files 😅

So with the current implementation, If I import the component as follows everything works:
```js
import CFDAccountManager from 'Components/cfd-account-manager';
```
But I can also import it directly, which in that case CSS won’t load:
```js
import CFDAccountManager from 'Components/cfd-account-manager/cfd-account-manager';
```

In my option, It would be safer to attach the CSS to the component itself instead of the barrel file.

Reply:
You're completely right, it is the safer approach, I just wanted to stick to the structure that we have and we can discuss it with the team whether we need to do it or not. Let's just stick to the shorter import until then :D .